usaspending_api/spending/v2/filters/fy_filter.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+
+from usaspending_api.common.exceptions import InvalidParameterException
+
+
+def fy_filter(fy, now):
+
+    # Define fiscal reporting quarters
```

Comment:
These fiscal quarter bounds shouldn't be calculated manually. There are functions available to do this automatically based on the date.

Reply:
I'm not aware of those functions.